### PR TITLE
test: Update SDK contact management tests

### PR DESCRIPTION
**Pull request by alexandra-marin**
#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/1978/update-sdk-contact-management-tests-to-the-new-framework
#### Testing instructions  
npm run test:e2e

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: test/e2e/code/steps/contacts.js [hunk 1]
```diff
@@ -1,93 +1,112 @@
...
+    });
+
+    When('no profiles are found', async function() {
+        this.contact.notFound.should.be.true;
+    });
+
+    Then('the contact is added in my contacts', async function() {
+        await this.waitForObservable(() => this.contact.isAdded);
+        this.ice.contactStore.contacts[0].should.equal(this.contact);
+        // this.ice.contactStore.contacts.should.deep.include(this.contact); // this fails
```
Comment (alexandra-marin) [inline test/e2e/code/steps/contacts.js]: @anri-asaturov not sure why this assert fails. can you take a look please?

Comment (anri-asaturov) [reply test/e2e/code/steps/contacts.js]: You don’t need deep equality in here, contact store creates and reuses same instance for each existing contact

Comment (alexandra-marin) [reply test/e2e/code/steps/contacts.js]: fails without deep too

File: test/e2e/code/world.js [hunk 2]
```diff
@@ -47,6 +51,34 @@ class PeerioAppWorld {
...
         await this.ice.contactStore.currentUser.ensureLoaded();
     }
+
+    confirmPrimaryEmail = async (emailAddress) => {
+        const email = await waitForEmail(emailAddress, testConfig.primaryEmailConfirmSubject);
+        const url = testConfig.emailConfirmUrlRegex.exec(email.body)[1];
+        await getUrl(url);
+    }
+
+    createUser = async (username, email) => {
```
Comment (anri-asaturov) [inline test/e2e/code/world.js]: what's the reason to do this move?

Comment (anri-asaturov) [reply test/e2e/code/world.js]: Ok, i see. But i don't like this, I think we should have more universal 'create other user' step. And world.js should be as small as possible. I'll think about a way to do this if you want.

Comment (anri-asaturov) [reply test/e2e/code/world.js]: Try to think about 'what happens when we write chat tests, file sharing tests, etc'

**Pull request merged**